jetbot/bot.py

```diff
@@ -496,6 +496,49 @@ async def channels(ctx: commands.Context):
                                 channel = await channel_methods[channel_config.get("type", "text")](category)(channel_config["name"])
                                 new.append(channel)
 
+                            # Create or update forum tags if it's a forum channel and has available_tags
+                            if isinstance(channel, ForumChannel) and "available_tags" in channel_config:
+                                print("Processing forum tags for... " + channel_config["name"])
+                                # Get existing tags
+                                existing_tags = [tag.name for tag in channel.available_tags]
+                                existing_tag_objects = {tag.name: tag for tag in channel.available_tags}
+
+                                # Validate total number of tags
+                                new_tags = [tag for tag in channel_config["available_tags"] if tag not in existing_tags]
+                                if len(existing_tags) + len(new_tags) > 20:
```

Comment:
This logic doesn't seem right, surely you just need to check the length of `channel_config["available_tags"]`, as you'll be removing some that're in `existing_tags`?